code/modules/hydroponics/grown/cereals.dm

```diff
@@ -25,6 +25,8 @@
 	grind_results = list(/datum/reagent/consumable/flour = 0)
 	tastes = list("wheat" = 1)
 	distill_reagent = /datum/reagent/consumable/ethanol/beer
+	slot_flags = ITEM_SLOT_MASK
+	worn_icon = 'icons/mob/clothing/mask.dmi'
```

Comment:
Why put thesein `'icons/mob/clothing/mask.dmi'` instead of `'icons/mob/clothing/head/hydroponics.dmi'`?

Reply:
Figured I'd put it in the same place as cigs as they're the same kind of thing. I don't feel strongly about it though, and I can move them there if you think that's better.

Reply:
I'd prefer them in the hydroponics dmi personally, same place as roses/flowers

Reply:
Seconded